README.md

```diff
@@ -156,6 +158,15 @@ Download the wheel file of the [latest release](https://github.com/fdw/rofimoji/
 Afterwards, there should be a `rofimoji` on your `$PATH`.
 This also installs the python dependency `configargparse`.
 
+## MacOS support
```

Comment:
This should probably be under `dependencies`, maybe merged with the paragraph there?

In any case, it doesn't fit under `Installation`.

Reply:
Done :) let me know if you prefer I merge the sections or keep it as is